docker/python2/official/Dockerfile

```diff
@@ -0,0 +1,55 @@
+# *****************************************************************************
+#
+# Copyright (c) 2019, the Perspective Authors.
+#
+# This file is part of the Perspective library, distributed under the terms of
+# the Apache License 2.0.  The full license can be found in the LICENSE file.
+#
+
+FROM python:2.7
+RUN apt-get update
+RUN apt-get -y install apt-transport-https libtbb-dev rapidjson-dev sudo
+RUN apt-get -y remove python
+
+RUN wget https://cmake.org/files/v3.15/cmake-3.15.4-Linux-x86_64.sh -q
+RUN mkdir /opt/cmake
+RUN printf "y\nn\n" | sh cmake-3.15.4-Linux-x86_64.sh --prefix=/opt/cmake > /dev/null
+RUN rm -fr cmake*.sh /opt/cmake/doc
+RUN rm -fr /opt/cmake/bin/cmake-gui
+RUN rm -fr /opt/cmake/bin/ccmake
+RUN rm -fr /opt/cmake/bin/cpack
+RUN ln -s /opt/cmake/bin/cmake /usr/local/bin/cmake
+RUN ln -s /opt/cmake/bin/ctest /usr/local/bin/ctest
+
+RUN python2.7 -m pip install numpy pandas cython 'mock==2.0.0' codecov nose2 flake8 pytest pytest-cov traitlets ipywidgets faker psutil zerorpc 'backports.shutil_which'
+
+# install boost
+RUN wget https://dl.bintray.com/boostorg/release/1.71.0/source/boost_1_71_0.tar.gz >/dev/null 2>&1
+RUN tar xfz boost_1_71_0.tar.gz
+# https://github.com/boostorg/build/issues/468
+RUN cd boost_1_71_0 && ./bootstrap.sh
+RUN cd boost_1_71_0 && ./b2 -j8 --with-program_options --with-filesystem --with-system install 
```

Comment:
Would be really nice if the diffs between these `Dockerfile` used a base image, which would also alleviate the build times.